<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grades</class>
 <widget class="QDialog" name="grades">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{font-size:30px;}</string>
   </property>
   <property name="text">
    <string>Grades</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>621</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_20">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt</string>
        </property>
        <property name="text">
         <string>Subject</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_21">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt</string>
        </property>
        <property name="text">
         <string>Grade</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_22">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt</string>
        </property>
        <property name="text">
         <string>Rating</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt</string>
        </property>
        <property name="text">
         <string>Programming</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt</string>
        </property>
        <property name="text">
         <string>Maths</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt</string>
        </property>
        <property name="text">
         <string>Discrete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt</string>
        </property>
        <property name="text">
         <string>Circuits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt</string>
        </property>
        <property name="text">
         <string>Solid state</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_17">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt</string>
        </property>
        <property name="text">
         <string>TOTAL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_19">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>550</y>
     <width>287</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_24">
      <property name="styleSheet">
       <string notr="true">font-size: 18pt</string>
      </property>
      <property name="text">
       <string>Percentage:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_23">
      <property name="styleSheet">
       <string notr="true">font-size: 18pt</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
